vulnerable_people_form/form_pages/check_your_answers.py

```diff
@@ -16,7 +16,7 @@ def get_check_your_answers():
     exclude_answers = None
 
     if current_app.is_tiering_logic_enabled:
-        if is_very_high_plus_shielding_without_basic_care_needs_answer():
+        if is_shielding_without_basic_care_needs_answer():
             return redirect(append_querystring_params("/basic-care-needs"))
         if get_location_tier() == PostcodeTier.VERY_HIGH.value:
```

Comment:
i think this if statement should check if they are not shielding then exclude basic care needs
`
if get_shielding_advice() == ShieldingAdvice.NOT_ADVISED_TO_SHIELD.value:`